### DISCUSSION TITLE
Turning on usd array attributes in maya

### DISCUSSION THREAD

**Discussion by adamdewhirst** [Autodesk/maya-usd#1557] [Q&A]
Hello I'm trying to get instancing to work in maya with usd

right now i cant seem to turn on the instanceable tag, because i cant see all the usd array attributes

following this: https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2022/ENU/Maya-USD/files/USD-for-Maya/USD-in-the-Maya-Interface/GUID-5355AFE8-D39C-4950-A7BB-19407D59C626-html.html

however I'm seeing nothing in the show menu to reveal array attributes

unless its on by default and I'm missing something?

specifically look at a def prim - i can see Xformable, Imageable, but i can't see the metadata attributes

is there a reason this isn't showing up that I'm not aware of?

**Comment by seando-adsk** [comment]
What version of Maya are you using? That menu requires UFE v2 which is only in Maya 2022.

**Reply by adamdewhirst** [reply]
yup, using 2022

**Reply by seando-adsk** [reply]
Okay good. And just to be clear after you've loaded the MayaUsd plugin in the Attribute Editor **Show** Menu at the bottom you don't see a section "Universal Scene Description" with an item "Show Array Attributes"?

Are you using the MayaUsd plugin installed with Maya 2022 or did you build it yourself. If you built it yourself make sure to check your build log to ensure it correctly found UFE.
